os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -453,19 +523,19 @@ static int loading_thread(int argc, char *argv[])
 			bmdbg("Invalid arguments for reloading, argc %d\n", argc);
 			break;
 		}
-		ret = binary_manager_check_update_binary(argv[2]);
-		if (ret == BINMGR_OK) {
-			ret = binary_manager_reload(argv[2]);
-		}
+		/* [2] binary name for reloading */
+		ret = binary_manager_update(argv[2]);
```

Comment:
I think update and reload have similar process.
terminate related task/pthreads and load.
Are they possible to call the common function?

Reply:
Modified